Sources/DarkModeCore/UIView+DarkModeKit.m

```diff
@@ -29,8 +29,10 @@ - (void)dmTraitCollectionDidChange:(DMTraitCollection *)previousTraitCollection
   }];
   [self setNeedsLayout];
   [self setNeedsDisplay];
+#if __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_13_0
```

Comment:
wrapping this in the else block of if (@available(iOS 13.0, *)) does not work, so this is the only way I could think of to fix the warning.

Reply:
Looks like this flag is defined in internal header file: https://opensource.apple.com/source/CarbonHeaders/CarbonHeaders-18.1/AvailabilityInternal.h.auto.html

Any other opensource project is using it?

Reply:
https://github.com/forkingdog/FDStackView/blob/d507a47d185de1cb0cdd35cdeb28fcdd827d9859/FDStackView/FDStackView.h#L29

FDStackView

Reply:
I'm not sure why we need to do this. Is there a compiler warning? If that's the case, can we put these code into a else sentence of @available to resolve the warning?

Reply:
yes, we marked dm_updateDynamicColors and dm_updateDynamicImages as deprecated on iOS 13. I've tried putting in the else block, it works only for the warnings in Swift but not Objective-C.